src/builtins/BuiltinTypedArray.cpp

```diff
@@ -1157,6 +1145,82 @@ static Value builtinTypedArrayFindIndex(ExecutionState& state, Value thisValue,
     return Value(-1);
 }
 
+static Value builtinTypedArrayFindLast(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    Value predicate = argv[0];
+    Value thisArg = argc > 1 ? argv[1] : Value();
+    // Let O be ToObject(this value).
+    RESOLVE_THIS_BINDING_TO_OBJECT(O, TypedArray, find);
+    // validateTypedArray is applied to the this value prior to evaluating the algorithm.
+    TypedArrayObject::validateTypedArray(state, O);
+
+    // Let len be O.[[ArrayLength]].
+    int64_t len = static_cast<int64_t>(O->asTypedArrayObject()->arrayLength());
+
+    // If IsCallable(predicate) is false, throw a TypeError exception.
+    if (!predicate.isCallable()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, state.context()->staticStrings().TypedArray.string(), true, state.context()->staticStrings().findLast.string(), ErrorObject::Messages::GlobalObject_CallbackNotCallable);
```

Comment:
Good suggestion!
But in this case, `StaticStrings` is required only twice in this function.
So, I think that its better to refer `StaticStrings` whenever it is necessary instead of allocating a new variable(reference) and reusing it.

BTW, for other cases in which `StaticStrings` is frequently required, reference is reused to make the code length short like below.
https://github.com/Samsung/escargot/blob/b1aec3b7b9e33332ff62a4332f9df5816a23ed71/src/builtins/BuiltinTypedArray.cpp#L1633-L1637